### DISCUSSION TITLE
Build Mydumper to backup and restore mariadb 10.5 to 10.6

### DISCUSSION THREAD

**Discussion by mukul4u2005** [mydumper/mydumper#1454] [Q&A] [answered]
Hi  I followed the documentation to build and all steps done successfully. just one step :   

E: Package 'libmariadbclient-dev' has no installation candidate

so i used this command: apt-get install default-libmysqlclient-dev , its worked after that

But when I am running mydumper -V its showing:

root@leader:~/mydumper/mydumper# mydumper -V
mydumper v0.16.2-1, built against MySQL 8.0.36 with SSL support

But Mariadb showing version:

root@leader:~/mydumper/mydumper# mariadb -V
mariadb  Ver 15.1 Distrib 10.6.16-MariaDB, for debian-linux-gnu (x86_64) using  EditLine wrapper

Any suggestion, why its showing Mysql ?

**Comment by davidducos** [comment, marked as answer]
you forgot the `./` . If you check `which mydumper` you will see what you executing. To execute the one that you have just compiled you need to run `root@leader:~/mydumper/mydumper# ./mydumper -V`
